.github/workflows/ci_trie_benchmark.yml

```diff
@@ -0,0 +1,42 @@
+name: trie benchmark
+
+on:
+  workflow_call:
+  pull_request:
+    branches: ["**"]
+    paths:
+      - "crates/l2/**"
```

Comment:
this should be the trie

Reply:
Addressed [here](https://github.com/lambdaclass/ethrex/pull/2272/commits/db0fee68cf21c2327696524447327e53511cf464).